src/parser/priest/discipline/modules/spells/SinsOfTheMany.tsx

```diff
@@ -21,38 +21,42 @@ const SINS_OF_THE_MANY_FLOOR_BONUS = 0.03;
  * from the passive and from 6 onwards it only decreases 0.005.
  * Hence this map with the values for each Atonement count.
  */
-const BONUS_DAMAGE_MAP = {
```

Comment:
The object here is more legible for looking at target count to bonus damage mapping

Reply:
objects being indexed with ints is usually a pretty gangly practice imo. I'd almost rather make it into a function instead of a purely int indexed object. Mind you there are objects in these modules that are indexed by objects so /shrug

I can change it if you really think it should remain an object.